Sources/UITextFieldExtensions.swift

```diff
@@ -47,3 +47,43 @@ extension UITextField {
     }
 
 }
+
+
+/// MARK: Validations
+extension UITextField {
+    /// EZSE: Ways to validate by comparison
+    enum textFieldValidationOptions: Int {
+        case equalTo, greaterThan, greaterThanOrEqualTo, lessThan, lessThanOrEqualTo
+    }
+    
+    /// EZSE: Validation length of character counts in UITextField
+    func validateLength(ofCount count: Int, option: UITextField.textFieldValidationOptions) -> Bool {
```

Comment:
Validating password field of min length
Validate min length of a query before make a search